airflow/providers/celery/executors/celery_executor.py

```diff
@@ -368,8 +368,10 @@ def update_all_task_states(self) -> None:
             if state:
                 self.update_task_state(key, state, info)
 
-    def change_state(self, key: TaskInstanceKey, state: TaskInstanceState, info=None) -> None:
-        super().change_state(key, state, info)
+    def change_state(
+        self, key: TaskInstanceKey, state: TaskInstanceState, info=None, remove_running=True
+    ) -> None:
+        super().change_state(key, state, info, remove_running=remove_running)
```

Comment:
Same here no? If Airflow version is less than 2.10, it will not recognize `remove_running`?

Reply:
This is actually using the default and just plumbing it through, it was to make the static checks happy because they compare the stub against the super class. I'm not sure the best way to get around this one. Perhaps just no-qa, since it's actually fine and desirable to just use the default here, no real need to plumb it through.

Reply:
Went with a try/catch here as well. Thoughts?

Reply:
LGTM

Reply:
yeah, I like that.